test/unit/core/client.test.ts

```diff
@@ -22,6 +22,7 @@ describe('client', function () {
       logger: nullLogger(),
       environment: null
     })
+    client.configure()
```

Comment:
We have to make sure to call `configure` any time we create a new instance of `Honeybadger`, otherwise `__store` will not be initialized. I think this is OK since `configure` is necessary to set the apiKey. If a new instance is created with `Honeybadger.factory()`, then it will be OK.

Reply:
Couldn't we just set `this.__store = opts.store || new DefaultStore()` when initializing the client, but let `configure` override it?

Reply:
Yeah, that's what I had at first, but it seems that `configure()` is a required method call for Honeybadger to work (need to at least set the `apiKey`). It's just that we don't always call `configure()` in our tests. I can add it if you think it may raise some issues.